common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -387,6 +387,16 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
+  val prebidS2SOzoneBidder: Switch = Switch(
```

Comment:
Is `ozone` the code name?

Reply:
I'm not sure. The technical project is called the-ozone-project (like in the URL), but the internal codename is 'whitepowder' #suchconfusion